[2023-02-04 09:02:03,009] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2023-02-04T09:01:58.402810+00:00 [queued]>
[2023-02-04 09:02:03,019] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2023-02-04T09:01:58.402810+00:00 [queued]>
[2023-02-04 09:02:03,020] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 09:02:03,020] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-02-04 09:02:03,021] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 09:02:03,040] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): insert_data> on 2023-02-04 09:01:58.402810+00:00
[2023-02-04 09:02:03,044] {standard_task_runner.py:52} INFO - Started process 10472 to run task
[2023-02-04 09:02:03,047] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-rates-test', 'insert_data', 'manual__2023-02-04T09:01:58.402810+00:00', '--job-id', '254', '--raw', '--subdir', 'DAGS_FOLDER/yandex_dag_test.py', '--cfg-path', '/tmp/tmpg6hlencu', '--error-file', '/tmp/tmpa_bzchoz']
[2023-02-04 09:02:03,047] {standard_task_runner.py:80} INFO - Job 254: Subtask insert_data
[2023-02-04 09:02:03,116] {task_command.py:371} INFO - Running <TaskInstance: yandex-rates-test.insert_data manual__2023-02-04T09:01:58.402810+00:00 [running]> on host 39df186e1726
[2023-02-04 09:02:03,201] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-rates-test
AIRFLOW_CTX_TASK_ID=insert_data
AIRFLOW_CTX_EXECUTION_DATE=2023-02-04T09:01:58.402810+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-04T09:01:58.402810+00:00
[2023-02-04 09:02:03,214] {logging_mixin.py:115} INFO - rate_date:  2023-02-04
[2023-02-04 09:02:03,215] {logging_mixin.py:115} INFO - value_:  23510.60869599999932
[2023-02-04 09:02:03,225] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-02-04 09:02:03,252] {python.py:173} INFO - Done. Returned value was: None
[2023-02-04 09:02:03,266] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-rates-test, task_id=insert_data, execution_date=20230204T090158, start_date=20230204T090203, end_date=20230204T090203
[2023-02-04 09:02:03,300] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-04 09:02:03,341] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
